feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -127,6 +128,18 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	dc.Interface(i2.GetName()).Replace(t, i2)
 }
 
+//Configure network instance
+func configureNetworkInstance(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	d := &telemetry.Device{}
+	ni1 := d.GetOrCreateNetworkInstance(NetworkInstance)
+	ni1.Type = telemetry.E_NetworkInstanceTypes_NETWORK_INSTANCE_TYPE(*ygot.Int64(1))
```

Comment:
Use the enum for the default network instance type, as opposed to a magic value?